__snapshots__/e2e.spec.js

```diff
@@ -53,18 +51,12 @@ Fix the error in your code and re-run your tests.
   │ Pending:      0                                                                                │
   │ Skipped:      0                                                                                │
   │ Screenshots:  0                                                                                │
-  │ Video:        true                                                                             │
+  │ Video:        false                                                                            │
   │ Duration:     X seconds                                                                        │
   │ Spec Ran:     e2e/compile-error.js                                                             │
   └────────────────────────────────────────────────────────────────────────────────────────────────┘
 
 
-  (Video)
-
-  -  Started processing:  Compressing to 32 CRF                                                     
-  -  Finished processing: /[cwd]/cypress/videos/e2e/compile-error.js.mp4                  (X second)
```

Comment:
was causing problem with Cypress 4.3.0 and updating the snapshot quickly ran into a problem running locally